SpringProject/src/it/java/com/gradproject2019/user/UserControllerIT.java

```diff
@@ -58,7 +62,7 @@ public void shouldReturn200AndSaveUserInDatabase() {
         HttpEntity<UserRequestDto> request = new HttpEntity<>(userRequestDto);
 
         ResponseEntity<String> response = postUser(request);
-        User retrievedUser = userRepository.findAll().get(1);
+        User retrievedUser = userRepository.findAll().get(3);
```

Comment:
3 seems like a bit of a magic number here - what significance does it have to this test?

Reply:
it's just because i created 2 extra users for testing so now there are 3 in total. There is probably a better way to test this that won't break if new users are added

Reply:
removed addition users so this does not apply any more!